### ISSUE TITLE
TST will sporadically close and the TST icon in the toolbar will disappear.

### ISSUE THREAD

**Issue by rileyjw90** [piroor/treestyletab#2891]
Since TST updated a few days ago, it will sporadically close and the icon will disappear from the toolbar. When I click "Customize Toolbar" in the firefox menu, it is not there. To get it to reappear, I have to go to Extensions, disable TST, and then re-enable. 

I cannot say how to reproduce this issue, as it is sporadic and closes when I am away from the computer for a period of time, so I have not directly witnessed it. Perhaps it has to do with how long firefox is open but idle? I never close firefox unless I'm running CCleaner, so it is always being actively used or hanging out in the background.

 * Platform (OS): Windows 10 Pro (version 1909, OS build 18363.1500)
 * Version of Firefox: 88.0.1 (64-bit)
 * Version (or revision) of Tree Style Tab: 3.8.0

**Comment by irvinm** [comment]
@rileyjw90 is this still happening for you?  Have you isolated any patterns for when it happens?

I'm also wondering if you were to check the browser console after this happens to see if you see any errors or relevant messages.

**Comment by piroor** [comment]
This kind problem is hard to be solved without deep investigation with debugger on my local environment. Please add comments if you successfully narrow down the steps to reproduce the problem. (Of course debugging by yourself and writing a patch are both welcome!)

**Comment by irvinm** [comment]
@rileyjw90 any new updates?  Have you tried to see if 3.8.2 is any better for you?

**Comment by irvinm** [comment]
@rileyjw90 just a friendly reminder.

**Comment by rileyjw90** [comment]
Hi! Apologies for the delay. I completely removed TST and then reinstalled it, and the issue seemed to resolve itself. It happened a few more times over a period of a couple weeks and removing/reinstalling would keep it at bay for several days in a row, but since I approved the newer update of TST, it has not been an issue at all. I'm not sure what was going on (maybe an issue with another extension, or perhaps my pi-hole was interfering somehow, since I set it up not long before the issue presented itself for me). It seems to be an issue specific to my particular setup and fortunately I am no longer affected by it. Thank you for the responses!
